sources/models/shared-defaults/oci-hooks.toml

```diff
@@ -1,13 +1,13 @@
 [settings.oci-hooks]
 log4j-hotpatch-enabled = false
 
-[metadata.settings.oci-hooks]
-affected-services = ["oci-hooks"]
+[metadata.settings.oci-hooks.log4j-hotpatch-enabled]
+affected-services = ["log4j-hotpatch-enabled"]
 
-[services.oci-hooks]
-configuration-files = ["oci-hooks"]
-restart-commands = []
+[services.log4j-hotpatch-enabled]
+configuration-files = ["log4j-hotpatch-enabled-log-message"]
+restart-commands = ["systemctl restart deprecation-warning@log4j-hotpatch-enabled.timer"]
 
-[configuration-files.oci-hooks]
-path = "/etc/shimpei/shimpei-hooks.json"
-template-path = "/usr/share/templates/oci-default-hooks-json"
```

Comment:
Can we keep these defaults around for the top-level `oci-hooks` settings? Just in case we add another hook down the road. That seems like it'll be simpler.

If we drop these defaults, then I think we'd need a downward migration to put them back for older versions. Otherwise, the hooks won't work properly on downgrade because the new version won't have set these up in the datastore.

Reply:
The downgrade issue I'm thinking of would happen when launching a brand-new 1.15.0 instance and downgrading to 1.14.3.

An upgrade/downgrade cycle from 1.14.3 wouldn't trigger it, which is what it looks like you tested.

Reply:
I'll test to confirm the downgrade cycle you discussed.